palindrome.c

```diff
@@ -0,0 +1,38 @@
+/*Jinoo Hong*/
+/*Write a program `palindrome.c` using recursion to test if a word/phrase is a palindrome (something that reads the same if the letters are reversed, such as racecar).*/
```

Comment:
Nice job! The only thing is you have all your intermediate steps printed out, which I don't know that I'd necessarily include. You also didn't handle capitals! (so Hannah, which is a palindrome, wasn't handled as such)